features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml

```diff
@@ -29,7 +29,6 @@
       <import feature="org.eclipse.wst.web_ui.feature" version="3.7.1.v201602111638" match="greaterOrEqual"/>
    </requires>
 
-   <!-- appengine-plugins-core -->
```

Comment:
I think it's okay to leave this comment here. I was confused once why we don't have this `com.google.cloud.tools.appengine` in my workbench.